app/assets/javascripts/application.js

```diff
@@ -16,3 +16,13 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+def reverse_string(string)
+  new_string = []
+  i = string.length-1
+  while i >= 0
+    new_string.push(string[i])
```

Comment:
Identifier 'new_string' is not in camel case.<br>Expected '{' and instead saw 'new_string'.<br>Missing semicolon.<br>'new_string' is not defined.<br>'string' is not defined.<br>'i' is not defined.